README.md

```diff
@@ -524,17 +524,20 @@ The custom options are:
 
 - `documentTypes` (object)
 
-  The list of document types visible to the user can be filtered by using the `documentTypes` option. The default value for each document type is `true`. If `documentTypes` only includes one document type, users will not see either the document selection screen or the country selection screen and instead will be taken directly to the capture screen.
+  The list of document types visible to the user can be filtered by using the `documentTypes` option. The default value for each document type is `true`.
 
 - `country` (string)
 
   Document country can be specified per document type. The `country` configuration for a document type allows you to specify the issuing country of the document as a string containing a 3-letter ISO 3166-1 alpha-3 country code.
 
+  If `documentTypes` only includes one document type with a country value, users will not see either the document selection and instead will be taken directly to the capture screen.
+
+  ⚠️ **Note**: the `null` value is deprecated and has no effect.
   If a document country is specified for a document type, or is passed as `null`, the country selection screen is not displayed to the end user.
```

Comment:
should be remove this line then? @Phoebe-B

Reply:
Moving it below as suggested by @Phoebe-B